From 82e092e9debce16f048b4fe0f38265bc8d80f63d Mon Sep 17 00:00:00 2001
From: William Brown <firstyear@redhat.com>
Date: Thu, 28 Sep 2017 09:11:00 +1000
Subject: [PATCH] Ticket 49378 server init fails

Bug Description:  We used our own target for DS installation, but
we should just use multi-user like anything else.

Fix Description:  Change service template to multi-user. This should
be a seamless upgrade to most consumers.

https://pagure.io/389-ds-base/issue/49378

Author: wibrown

Review by: mreynolds (Thanks!)

(cherry picked from commit e9ad5f5aca64f65fa2c9b2dc5132b0dacf131c99)
---
 wrappers/systemd.template.asan.service.in | 2 +-
 wrappers/systemd.template.service.in      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wrappers/systemd.template.asan.service.in b/wrappers/systemd.template.asan.service.in
index 1fe321ccb..52681f632 100644
--- a/wrappers/systemd.template.asan.service.in
+++ b/wrappers/systemd.template.asan.service.in
@@ -36,5 +36,5 @@ ExecStart=@sbindir@/ns-slapd -D @instconfigdir@/slapd-%i -i @localstatedir@/run/
 .include @initconfigdir@/@package_name@.systemd
 
 [Install]
-WantedBy=dirsrv.target
+WantedBy=multi-user.target
 
diff --git a/wrappers/systemd.template.service.in b/wrappers/systemd.template.service.in
index 30b9e4b78..0d88900b6 100644
--- a/wrappers/systemd.template.service.in
+++ b/wrappers/systemd.template.service.in
@@ -40,5 +40,5 @@ ExecStart=@sbindir@/ns-slapd -D @instconfigdir@/slapd-%i -i @localstatedir@/run/
 .include @initconfigdir@/@package_name@.systemd
 
 [Install]
-WantedBy=dirsrv.target
+WantedBy=multi-user.target
 
-- 
2.13.6

